beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/filter/LocationOnlyUpdateListenerEventFilter.java

```diff
@@ -44,6 +55,11 @@ public boolean isFiltered(ListenerEvent listenerEvent, LifecycleEventType lifecy
   }
 
   private boolean isLocationSame(String oldLocation, String location) {
-    return location == null || oldLocation == null || oldLocation.equals(location);
+    if (location == null || oldLocation == null) {
+      return true;
+    }
+    String normalizedOldLocation = locationNormalizer.normalize(oldLocation);
+    String normalizedLocation = locationNormalizer.normalize(location);
+    return normalizedOldLocation.equals(normalizedLocation);
```

Comment:
We should expand in follow up PR to check that location doesn't start with oldLocation and vica versa.

old: `s3://foo/bar`
new: `s3:/foo`

Should be considered same.

Reply:
I can't say right now if it's a good idea.